upf/pfcp.c

```diff
@@ -1095,11 +1110,11 @@ decode_bit_rate (u8 * data, u16 length, void *p)
 {
   pfcp_bit_rate_t *v = p;
 
-  if (length < 8)
+  if (length < 10)
     return PFCP_CAUSE_INVALID_LENGTH;
 
-  v->ul = get_u32 (data);
-  v->dl = get_u32 (data);
+  v->ul = get_u40 (data);
```

Comment:
This is causing build error
```
src/vpp/src/plugins/upf/pfcp.c:1116:11: error: shift count >= width of type [-Werror,-Wshift-count-overflow]
```
apparently due to `data` being `u8 *` upon `(V)[0] << 32` etc

Reply:
Ref: https://github.com/travelping/upg-vpp/runs/2944778616?check_suite_focus=true#step:3:3127

Reply:
fixed, but I really wonder why this does not happen for u24 getter.